applications/maplab-server-node/src/maplab-server-node.cc

```diff
@@ -155,22 +167,32 @@ void MaplabServerNode::start() {
     return;
   }
 
+  if (load_previous_state) {
```

Comment:
why this duplicated logic? Could we not only check whether the `initial_map_path_` is empty?

Reply:
How would you differ between a regular startup and a respawn then?